djantic/main.py

```diff
@@ -172,6 +173,8 @@ def get(self, key: Any, default: Any = None) -> Any:
             attr = list(attr.all())
         elif outer_type_ == int and issubclass(type(attr), Model):
             attr = attr.id
+        elif inspect.isclass(type(attr)) and issubclass(type(attr), Enum):
```

Comment:
Unrelated to the nullable related field change: the `tests/test_fields.py::test_enum_choices` test was failing (even on `main` branch. This should fix that.

Test fail:
```
FAILED tests/test_fields.py::test_enum_choices - pydantic_core._pydantic_core.ValidationError: 2 validation errors for PreferenceSchema
preferred_food
  Input should be 'ba' or 'ap' [type=enum, input_value=<FoodChoices.BANANA: 'ba'>, input_type=FoodChoices]
    For further information visit https://errors.pydantic.dev/2.7/v/enum
preferred_group
  Input should be 1 or 2 [type=enum, input_value=<GroupChoices.GROUP_1: 1>, input_type=GroupChoices]
    For further information visit https://errors.pydantic.dev/2.7/v/enum
```

Reply:
Very nice.